tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -845,6 +845,29 @@ class Cl {}
             ['import_symbols' => true],
         ];
 
+        yield 'shorten relative reference to already imported, direct short name' => [
```

Comment:
this test is not needed /w https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/commit/94a29364ce6200f2dc0b6a7b6270269832fe68a3

Reply:
The impl. basically enforce one format, but still a test case is good - 👍